draft-ietf-trans-rfc6962-bis.md

```diff
@@ -2067,15 +2065,15 @@ the log, proving violation of the append-only property.
 
 ## Deterministic Signatures    {#deterministic_signatures}
 
-Logs are required to use deterministic signatures for the following reasons:
+Logs are encouraged to use deterministic signatures to reduce the risk of
+tracking clients: clients that gossip STHs or report back SCTs can be tracked
+or traced if a log was to produce multiple STHs or SCTs with the same timestamp
+and data but different signatures.
 
-* Using non-deterministic ECDSA with a predictable source of randomness means
-  that each signature can potentially expose the secret material of the signing
-  key.
-
-* Clients that gossip STHs or report back SCTs can be tracked or traced if a log
-  was to produce multiple STHs or SCTs with the same timestamp and data but
-  different signatures.
+Note that deterministic signatures can be provided without the use of a
+signature scheme that's designed for deterministic signing: The signature bytes
+for each SCT or STH could be stored and served when the same certificate is
+submitted or the same STH requested, rather than re-signing the same data.
```

Comment:
Done - with one modification, changing MUST to a SHOULD because I really don't want to require it yet, we don't really know how gossip will take place and if it'll indeed introduce a new tracking/fingerprinting vector.